### ISSUE TITLE
User-Agent is localized

### ISSUE THREAD

**Issue by onmyway133** [Alamofire/Alamofire#1269]
Hi, I see that `User-Agent` in `Manager` https://github.com/Alamofire/Alamofire/blob/master/Source/Manager.swift#L64 is localized

because of this 

```swift
let os = NSProcessInfo.processInfo().operatingSystemVersionString
```

> The operating system version string is human readable, localized, and is appropriate for displaying to the user. This string is not appropriate for parsing.

is this good?

**Comment by cnoon** [comment]
Great question @onmyway133! We'll have to do some investigation here and get back to you.

**Comment by cnoon** [comment]
cc @jshier, @AnthonyMDev, @kcharwood

**Comment by jshier** [comment]
Probably not a good idea, as Apple's docs explicitly call out not using the vale for parsing. Manually creating the version string from operatingSystemVersion seems like a better solution.

**Comment by cnoon** [comment]
I agree @jshier. I'll work on putting together a fix.

**Comment by bgerstle** [comment]
We've discovered this as well in our own internal attempts to set headers w/ the current OS version. Has anyone had success actually getting `NSProcessInfo.operatingSystemVersionString` to output a string with a different locale?  I've tried using Xcode scheme language options and the launch arguments (`-AppleLocale` and `-AppleLanguages`) to no avail.  Of course I also tried changing my Mac and iOS simulator language & region settings with no luck.

**Comment by onmyway133** [comment]
This is my temp solution

```swift
public static let osInfo: (version: String, build: String) = {
    let version = UIDevice.currentDevice().systemVersion

    var build = ""
    let os = NSProcessInfo.processInfo().operatingSystemVersionString
    let regex = try? NSRegularExpression(pattern: "\\(.*\\)", options: [])
    if let range = regex?.firstMatchInString(os, options: [], range: NSRange(location: 0, length: os.characters.count))?.range {
      let parts = (os as NSString).substringWithRange(range)
        .replace("(", with: "")
        .replace(")", with: "")
        .split(" ")

      if parts.count == 2 {
        build = parts[1]
      }
    }

    return (version: version, build: build)
  }()
```

**Comment by cnoon** [comment]
I'm going to close this issue out since @jshier has put together a fix in #1292. Please redirect all future comments to #1292.

Cheers. 🍻
